setup.py

```diff
@@ -14,7 +14,7 @@
 import warnings
 
 TORCH_MIN_VER = '1.5.0'
-TORCH_MAX_VER = '1.12.1'
+TORCH_MAX_VER = '1.16.1'
```

Comment:
Please don't bump torch maximum version as this require thorough testing on our side before updating, also the latest torch version is 1.13